hydra/_internal/core_plugins/basic_sweeper.py

```diff
@@ -54,14 +56,15 @@ class BasicSweeper(Sweeper):
     Basic sweeper
     """
 
-    def __init__(self, max_batch_size: Optional[int]) -> None:
+    def __init__(self, max_batch_size: Optional[int], params: Dict[str, str]) -> None:
```

Comment:
Is it possible that there is some downstream code that uses or inherits from `BasicSweeper`? If so, we could supply a default value for `params` so that this change will be less breaking to callers of `BasicSweeper.__init__`.
Here's what I'm thinking:
```python
def __init__(self, max_batch_size: Optional[int], params: Optional[Dict[str, str]] = None) -> None:
    if params is None:
        params = {}
```

Reply:
sure - we can just make it default to an empty dict right?